scripts/services/indexer/test_indexer

```diff
@@ -4,12 +4,14 @@ sleep 10
 result=$(PGPASSWORD=password psql -h $POSTGRES_HOST_IP -U postgres -d postgres -t -c "SELECT COUNT(*) FROM public.transactions;")
 result=$(echo $result | xargs)
 if (( result >= 1 )); then
-  http_status=$(curl -s -o /dev/null -w "%{http_code}" "http://localhost:8085/console")
-  if [ "$http_status" -eq 200 ]; then
+  response=$(curl -s -X POST -H "Content-Type: application/json" -d '{"query":"query {user_transactions { block_height } }"}' "http://localhost:8085/v1/graphql")
```

Comment:
Would be better to move this into a crate at some point.

Reply:
You prefer to implement it in Rust?